pkg/volume/util/subpath/subpath_linux.go

```diff
@@ -224,8 +226,13 @@ func doBindSubPath(mounter mount.Interface, subpath Subpath) (hostPath string, e
 	kubeletPid := os.Getpid()
 	mountSource := fmt.Sprintf("/proc/%d/fd/%v", kubeletPid, fd)
 
-	// Do the bind mount
+	info, err := hostutil.NewHostUtil().FindMountInfo(subpath.Path)
```

Comment:
is there any way to add a ut for this?

Reply:
so, I had no idea how to test the FindMountInfo method, because this method is based on some function that already has good [tests](https://github.com/kubernetes/kubernetes/blob/dbc2b0a5c7acc349ea71a14e49913661eaf708d2/staging/src/k8s.io/mount-utils/mount_helper_unix_test.go#L38). Therefore, I little modified the test for doBindSubPath and added a fake getter. How you look at it?